hive-plugins/src/main/java/co/cask/plugin/etl/batch/sink/HiveSinkConfig.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.plugin.etl.batch.sink;
+
+import co.cask.cdap.api.annotation.Description;
+import co.cask.cdap.api.annotation.Name;
+import co.cask.plugin.etl.batch.HiveConfig;
+
+import javax.annotation.Nullable;
+
+/**
+ * Configurations for Hive batch sink
+ */
+public class HiveSinkConfig extends HiveConfig {
+  @Name(Hive.FILTER)
+  @Description("Hive expression filter for write provided as a JSON Map of key value pairs that describe all of the " +
+    "partition keys and values for that partition. For example if the partition column is 'type' then this property " +
+    "should specified as:" +
+    "\"{\"type\":\"typeOne\"}\"." +
+    "To write multiple partitions simultaneously you can leave this empty, but all of the partitioning columns must " +
```

Comment:
can I have part of the partition specified here and the other part come from the record?  For example if I'm partitioned by type and date, can type come from this setting and date come from the record?